src/Fable.Cli/Main.fs

```diff
@@ -971,6 +976,10 @@ let private checkRunProcess (state: State) (projCracked: ProjectCracked) (compil
             | _, exeFile -> exeFile, runProc.Args
 
         if Option.isSome state.Watcher then
+            printfn "cliArgs.RunProcessEnv: %A" cliArgs.RunProcessEnv
```

Comment:
Is this still required?

Reply:
Probably not, I think it is me debugging something (if it me who added this code).

I think you also mentioned that for Vite we should not write directly to the console as it will mess up communication between processes.

Reply:
Well in `Fable.Cli` that wouldn't be a problem as that doesn't end up in the Vite plugin.
But since we introduced the logger, it feels wrong to ever use `printfn` again.
The lines are yours, I would remove them if we no longer need them.